test-network-k8s/docs/KUBERNETES.md

```diff
@@ -69,6 +69,36 @@ or:
 $ kind delete cluster
 ```
 
+## Rancher Desktop and k3s 
+
+In addition to KIND, the Kube Test Network runs on the k3s Kubernetes provided by [Rancher Desktop](https://rancherdesktop.io). 
+
+To run natively on k3s, skip the creation of a KIND cluster and:
+
+1. In Rancher's Kubernetes Settings:
+   1. Disable Traefik
+   2. Select the dockerd (moby) container runtime 
+   3. Increase Memory allocation to 8 GRAM 
+   4. Increase CPU allocation to 8 CPU 
+
+2. Reset Kubernetes 
+
+3. Initialize the Nginx ingress and cert-manager: 
+
+```shell
+export TEST_NETWORK_CLUSTER_RUNTIME=k3s
+
+./network cluster-init
```

Comment:
Just had another thought- is there an equivalent to `unkind` for this?

Reply:
Rancher desktop maintains a 1:1 "rancher" : "kubernetes is running" strategy.   So there's no real way to delete the rancher kube without going through the Rancher GUI and hitting the "reset Kubernetes" button.

Not to mention ... I couldn't find a verb that served as the inverse of "to rancher."  E.g.  `./network unrancher` doesn't have the same zing as `unkind`  -- I did try to find a good word for this, but abandoned the effort. 

I did make some changes to the teardown / `network down` script, to better scrub the kubernetes cluster of test network artifacts.

Reply:
Just wondering what undoes what `cluster-init` did really. I'm assuming that `network down` won't undo this?

```
✅ - Launching k3s ingress controller ...
✅ - Launching cert-manager ...
✅ - Waiting for cert-manager ...
✅ - Waiting for ingress controller ...
```

Maybe rename `network unkind` to `network clean`? Actually a generic `network prereqs` or some such which does the right thing depending on `TEST_NETWORK_CLUSTER_RUNTIME` might be better than having to pick between running `network kind` or `network cluster-init`?

Not worth holding this PR up for though.